dev/source/docs/gsoc-ideas-list.rst

```diff
@@ -9,17 +9,20 @@ This is a list of projects suggested by ArduPilot developers for `GSoC 2020 <htt
 - :ref:`Non-GPS navigation improvements using Intel RealSense cameras <common-vio-tracking-camera>`
 - :ref:`Object avoidance <common-object-avoidance-landing-page>` improvements for Multicopters and/or Rovers
 - `Optical flow <https://ardupilot.org/copter/docs/common-optical-flow-sensors-landingpage.html>`__ position hold performance improvements for multicopters
-- `Precision Landing <https://ardupilot.org/copter/docs/precision-landing-with-irlock.html>`__ accuracy improvements using `JeVois camera <http://www.jevois.org/>`__ (or similar) for multicopters
-- Automatic docking for cars and boats using `JeVois camera <http://www.jevois.org/>`__ (or similar)
+- `Precision Landing <https://ardupilot.org/copter/docs/precision-landing-with-irlock.html>`__ accuracy improvements for multicopters
```

Comment:
I'd prefer to keep this one as it was specifically talking about the jevois camera

Reply:
done